cordapp/src/main/kotlin/net/corda/plugins/CordappPlugin.kt

```diff
@@ -73,17 +71,41 @@ class CordappPlugin : Plugin<Project> {
         }
         jarTask.dependsOn(task)
 
-        val signTask = project.task("signCordappJar")
+        val optionalSignTask = createSigningTask(project, "singCordapp", false)
```

Comment:
sing -> sign

You might want to get into the habit of running `git diff | grep sing` until this work is done :)